media-sound/opensound/patches/opensound-4.2_git.patchset

```diff
@@ -80,16 +80,12 @@ index 8083e50..b9ab441 100644
      {
        strcpy (conf->platform, "i86pc");
      }
-@@ -30,5 +32,9 @@ check_sysdep (conf_t * conf, struct utsname *un)
+@@ -30,5 +32,5 @@ check_sysdep (conf_t * conf, struct utsname *un)
  static void
  add_kernel_flags (FILE * f)
  {
 -  fprintf (f, "CFLAGS=-O2 -D_KERNEL -D_KERNEL_MODE=1 -no-fpic\n");
-+  #if defined __x86_64__
-+    fprintf (f, "CFLAGS=-O2 -D_KERNEL -D_KERNEL_MODE=1\n");
-+  #else
-+    fprintf (f, "CFLAGS=-O2 -D_KERNEL -D_KERNEL_MODE=1 -no-fpic\n");
-+  #endif
++  fprintf (f, "CFLAGS=-O2 -D_KERNEL -D_KERNEL_MODE=1\n");
```

Comment:
Are you sure this doesn't break 32bit builds? I don't remember but usually -no-fpic is there to save a register being spilled by the compiler and avoid running out of them. Especially with gcc2 which is really picky...